src/hotspot/share/memory/arena.cpp

```diff
@@ -282,7 +282,7 @@ void Arena::set_size_in_bytes(size_t size) {
   if (_size_in_bytes != size) {
     ssize_t delta = size - size_in_bytes();
     _size_in_bytes = size;
-    MemTracker::record_arena_size_change(delta, _mem_tag);
+    MemTracker::record_arena_size_change(delta, 0, _mem_tag);
```

Comment:
Here the `allocated` param is passed as `0`, which means that the `allocated` amount won't change whereas the `requested` amount will change. So the calculation of overhead in memReporter.cpp (`size_t overhead = c->allocated() - c->requested();`) results in a negative number and changed to `unsigned` and becomes a very large positive number. This may describe why the Aren overhead in NMT report is more than 100%.